common/utils/backend.py

```diff
@@ -370,6 +372,25 @@ def get_next_num_run(self, peek: bool = False) -> int:
             self.active_num_run += 1
         return self.active_num_run
 
+    @staticmethod
+    def _is_run_dir(run_dir: str) -> bool:
```

Comment:
How do we check whether the run for the target directory is running now?

Reply:
we dont. That's why it causes issues.